src/server/wtr-config.js

```diff
@@ -188,7 +188,7 @@ export class WTRConfig {
 			//config.reporters.push(visualDiffReporter());
 
 			config.plugins ??= [];
-			config.plugins.push(visualDiff({ updateGoldens: this.#cliArgs.golden }));
+			config.plugins.push(visualDiff({ updateGoldens: golden, runSubset: filter?.length > 0 || grep?.length > 0 }));
```

Comment:
I think this ends up achieving the same thing. Could wrap it in `Boolean` if you want.

```suggestion
			config.plugins.push(visualDiff({ updateGoldens: golden, runSubset: filter || grep }));
```

Reply:
Yeah I think I'll wrap it in a boolean, feels weird passing in `null` or the array of values when I don't need to use it at all